---
author-meta:
- Halie M. Rando
- Casey S. Greene
- Michael P. Robson
- Simina M. Boca
- Nils Wellhausen
- Ronan Lordan
- Christian Brueffer
- Sadipan Ray
- Lucy D\'Agostino McGowan
- Anthony Gitter
- Anna Ada Dattoli
- Ryan Velazquez
- John P. Barton
- Jeffrey M. Field
- Bharath Ramsundar
- Adam L. MacLean
bibliography:
- content/manual-references.bib
- content/manual-references.json
date-meta: '2020-04-07'
header-includes: '<!--

  Manubot generated metadata rendered from header-includes-template.html.

  Suggest improvements at https://github.com/manubot/manubot/blob/master/manubot/process/header-includes-template.html

  -->

  <meta name="dc.format" content="text/html" />

  <meta name="dc.title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />

  <meta name="citation_title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />

  <meta property="og:title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />

  <meta property="twitter:title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />

  <meta name="dc.date" content="2020-04-07" />

  <meta name="citation_publication_date" content="2020-04-07" />

  <meta name="dc.language" content="en-US" />

  <meta name="citation_language" content="en-US" />

  <meta name="dc.relation.ispartof" content="Manubot" />

  <meta name="dc.publisher" content="Manubot" />

  <meta name="citation_journal_title" content="Manubot" />

  <meta name="citation_technical_report_institution" content="Manubot" />

  <meta name="citation_author" content="Halie M. Rando" />

  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />

  <meta name="citation_author_orcid" content="0000-0001-7688-1770" />

  <meta name="twitter:creator" content="@tamefoxtime" />

  <meta name="citation_author" content="Casey S. Greene" />

  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />

  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, Pennsylvania, United States of America" />

  <meta name="citation_author_orcid" content="0000-0001-8713-9213" />

  <meta name="twitter:creator" content="@GreeneScientist" />

  <meta name="citation_author" content="Michael P. Robson" />

  <meta name="citation_author_institution" content="Department of Computing Sciences, Villanova University, Villanova, Pennsylvania, United States of America" />

  <meta name="citation_author_orcid" content="0000-0002-4859-0033" />

  <meta name="citation_author" content="Simina M. Boca" />

  <meta name="citation_author_institution" content="Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, District of Columbia, United States of America" />

  <meta name="citation_author_orcid" content="0000-0002-1400-3398" />

  <meta name="citation_author" content="Nils Wellhausen" />

  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />

  <meta name="citation_author_orcid" content="0000-0001-8955-7582" />

  <meta name="citation_author" content="Ronan Lordan" />

  <meta name="citation_author_institution" content="Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104-5158, USA" />

  <meta name="citation_author_orcid" content="0000-0001-9668-3368" />

  <meta name="twitter:creator" content="@el_ronan" />

  <meta name="citation_author" content="Christian Brueffer" />

  <meta name="citation_author_institution" content="Department of Clinical Sciences, Lund University, Lund, Sweden" />

  <meta name="citation_author_orcid" content="0000-0002-3826-0989" />

  <meta name="twitter:creator" content="@cbrueffer" />

  <meta name="citation_author" content="Sadipan Ray" />

  <meta name="citation_author_institution" content="Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />

  <meta name="citation_author_institution" content="Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />

  <meta name="citation_author_orcid" content="0000-0002-9960-5768" />

  <meta name="citation_author" content="Lucy D\&#39;Agostino McGowan" />

  <meta name="citation_author_institution" content="Department of Mathematics and Statistics, Wake Forest University, Winston-Salem, North Carolina, United States of America" />

  <meta name="citation_author_orcid" content="0000-0001-7297-9359" />

  <meta name="twitter:creator" content="@LucyStats" />

  <meta name="citation_author" content="Anthony Gitter" />

  <meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin, United States of America" />

  <meta name="citation_author_institution" content="Morgridge Institute for Research, Madison, Wisconsin, United States of America" />

  <meta name="citation_author_orcid" content="0000-0002-5324-9833" />

  <meta name="twitter:creator" content="@anthonygitter" />

  <meta name="citation_author" content="Anna Ada Dattoli" />

  <meta name="citation_author_institution" content="Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />

  <meta name="citation_author_orcid" content="0000-0003-1462-831X" />

  <meta name="twitter:creator" content="@aadattoli" />

  <meta name="citation_author" content="Ryan Velazquez" />

  <meta name="citation_author_institution" content="Azimuth1, McLean, VA" />

  <meta name="citation_author_orcid" content="0000-0002-3655-3403" />

  <meta name="citation_author" content="John P. Barton" />

  <meta name="citation_author_institution" content="Department of Physics and Astronomy, University of California-Riverside, Riverside, California, United States of America" />

  <meta name="citation_author_orcid" content="0000-0003-1467-421X" />

  <meta name="twitter:creator" content="@_jpbarton" />

  <meta name="citation_author" content="Jeffrey M. Field" />

  <meta name="citation_author_institution" content="Department of Pharmacology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />

  <meta name="citation_author_orcid" content="000-0001-7161-7284" />

  <meta name="citation_author" content="Bharath Ramsundar" />

  <meta name="citation_author_institution" content="The DeepChem Project, https://deepchem.io/" />

  <meta name="citation_author_orcid" content="0000-0001-8450-4262" />

  <meta name="twitter:creator" content="@rbhar90" />

  <meta name="citation_author" content="Adam L. MacLean" />

  <meta name="citation_author_institution" content="Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America" />

  <meta name="citation_author_orcid" content="0000-0003-0689-7907" />

  <meta name="twitter:creator" content="@adamlmaclean" />

  <link rel="canonical" href="https://greenelab.github.io/covid19-review/" />

  <meta property="og:url" content="https://greenelab.github.io/covid19-review/" />

  <meta property="twitter:url" content="https://greenelab.github.io/covid19-review/" />

  <meta name="citation_fulltext_html_url" content="https://greenelab.github.io/covid19-review/" />

  <meta name="citation_pdf_url" content="https://greenelab.github.io/covid19-review/manuscript.pdf" />

  <link rel="alternate" type="application/pdf" href="https://greenelab.github.io/covid19-review/manuscript.pdf" />

  <link rel="alternate" type="text/html" href="https://greenelab.github.io/covid19-review/v/5ac11002700d669a9edb42a67d44261a2e5ac872/" />

  <meta name="manubot_html_url_versioned" content="https://greenelab.github.io/covid19-review/v/5ac11002700d669a9edb42a67d44261a2e5ac872/" />

  <meta name="manubot_pdf_url_versioned" content="https://greenelab.github.io/covid19-review/v/5ac11002700d669a9edb42a67d44261a2e5ac872/manuscript.pdf" />

  <meta property="og:type" content="article" />

  <meta property="twitter:card" content="summary_large_image" />

  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />

  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />

  <meta name="theme-color" content="#ad1457" />

  <!-- end Manubot generated metadata -->'
keywords:
- covid-19
- coronavirus
- pandemic
- viral infection
lang: en-US
manubot-clear-requests-cache: false
manubot-output-bibliography: output/references.json
manubot-output-citekeys: output/citations.tsv
manubot-requests-cache-path: ci/cache/requests-cache
title: 'SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics'
...






<small><em>
This manuscript
([permalink](https://greenelab.github.io/covid19-review/v/5ac11002700d669a9edb42a67d44261a2e5ac872/))
was automatically generated
from [greenelab/covid19-review@5ac1100](https://github.com/greenelab/covid19-review/tree/5ac11002700d669a9edb42a67d44261a2e5ac872)
on April 7, 2020.
</em></small>

<!-- include the Font Awesome library, per: https://fontawesome.com/start -->
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css">
[
[]{.fas .fa-info-circle .fa-lg} **This in progress manuscript is not intended for the general public.**<br>
This is a review paper that is authored by scientists for an audience of scientists to discuss research that is in progress.
If you are interested in guidelines on testing, therapies, or other issues related to your health, you should not use this document.
Instead, you should collect information from your local health department, the [CDC's guidance](https://www.cdc.gov/coronavirus/2019-ncov/index.html), or your own government.
]{.banner .lightred}

## Authors



+ **Halie M. Rando**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-7688-1770](https://orcid.org/0000-0001-7688-1770)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [rando2](https://github.com/rando2)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [tamefoxtime](https://twitter.com/tamefoxtime)<br>
  <small>
     Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
     · Funded by the Gordon and Betty Moore Foundation (GBMF 4552)
  </small>

+ **Casey S. Greene**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-8713-9213](https://orcid.org/0000-0001-8713-9213)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [cgreene](https://github.com/cgreene)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [GreeneScientist](https://twitter.com/GreeneScientist)<br>
  <small>
     Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Philadelphia, Pennsylvania, United States of America
     · Funded by the Gordon and Betty Moore Foundation (GBMF 4552)
  </small>

+ **Michael P. Robson**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-4859-0033](https://orcid.org/0000-0002-4859-0033)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [mprobson](https://github.com/mprobson)<br>
  <small>
     Department of Computing Sciences, Villanova University, Villanova, Pennsylvania, United States of America
  </small>

+ **Simina M. Boca**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-1400-3398](https://orcid.org/0000-0002-1400-3398)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [SiminaB](https://github.com/SiminaB)<br>
  <small>
     Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, District of Columbia, United States of America
  </small>

+ **Nils Wellhausen**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-8955-7582](https://orcid.org/0000-0001-8955-7582)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [nilswellhausen](https://github.com/nilswellhausen)<br>
  <small>
     Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
  </small>

+ **Ronan Lordan**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-9668-3368](https://orcid.org/0000-0001-9668-3368)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [RLordan](https://github.com/RLordan)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [el_ronan](https://twitter.com/el_ronan)<br>
  <small>
     Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104-5158, USA
  </small>

+ **Christian Brueffer**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-3826-0989](https://orcid.org/0000-0002-3826-0989)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [cbrueffer](https://github.com/cbrueffer)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [cbrueffer](https://twitter.com/cbrueffer)<br>
  <small>
     Department of Clinical Sciences, Lund University, Lund, Sweden
  </small>

+ **Sadipan Ray**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-9960-5768](https://orcid.org/0000-0002-9960-5768)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [rays1987](https://github.com/rays1987)<br>
  <small>
     Department of Systems Pharmacology & Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA; Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
  </small>

+ **Lucy D\'Agostino McGowan**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-7297-9359](https://orcid.org/0000-0001-7297-9359)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [LucyMcGowan](https://github.com/LucyMcGowan)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [LucyStats](https://twitter.com/LucyStats)<br>
  <small>
     Department of Mathematics and Statistics, Wake Forest University, Winston-Salem, North Carolina, United States of America
  </small>

+ **Anthony Gitter**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-5324-9833](https://orcid.org/0000-0002-5324-9833)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [agitter](https://github.com/agitter)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [anthonygitter](https://twitter.com/anthonygitter)<br>
  <small>
     Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin, United States of America; Morgridge Institute for Research, Madison, Wisconsin, United States of America
  </small>

+ **Anna Ada Dattoli**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-1462-831X](https://orcid.org/0000-0003-1462-831X)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [aadattoli](https://github.com/aadattoli)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [aadattoli](https://twitter.com/aadattoli)<br>
  <small>
     Department of Systems Pharmacology & Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
  </small>

+ **Ryan Velazquez**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-3655-3403](https://orcid.org/0000-0002-3655-3403)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [rdvelazquez](https://github.com/rdvelazquez)<br>
  <small>
     Azimuth1, McLean, VA
  </small>

+ **John P. Barton**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-1467-421X](https://orcid.org/0000-0003-1467-421X)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [johnbarton](https://github.com/johnbarton)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [_jpbarton](https://twitter.com/_jpbarton)<br>
  <small>
     Department of Physics and Astronomy, University of California-Riverside, Riverside, California, United States of America
  </small>

+ **Jeffrey M. Field**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [000-0001-7161-7284](https://orcid.org/000-0001-7161-7284)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [Jeff-Field](https://github.com/Jeff-Field)<br>
  <small>
     Department of Pharmacology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
  </small>

+ **Bharath Ramsundar**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-8450-4262](https://orcid.org/0000-0001-8450-4262)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [rbharath](https://github.com/rbharath)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [rbhar90](https://twitter.com/rbhar90)<br>
  <small>
     The DeepChem Project, https://deepchem.io/
  </small>

+ **Adam L. MacLean**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-0689-7907](https://orcid.org/0000-0003-0689-7907)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [alavendelm](https://github.com/alavendelm)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [adamlmaclean](https://twitter.com/adamlmaclean)<br>
  <small>
     Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America
  </small>



## Abstract {.page_break_before} 
Since late 2019, Coronavirus disease 2019 (COVID-19) has spread around the world, resulting in the declaration of a pandemic by the World Health Organization (WHO).
This infectious disease is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).
Research on the virus SARS-CoV-2 and the disease it causes is emerging rapidly through global scientific efforts.
The development of diagnostics, treatments, and vaccines will be critical to mitigating the impact of the virus.
Here we present a collaborative effort to organize and consolidate the rapidly emerging scientific literature related to SARS-CoV-2.
We present information about the virus in the context of what is known about related viruses and synthesize studies emerging about the diagnosis and treatment of COVID-19 alongside literature about related illnesses.
A broad scientific effort to understand this pandemic and related viruses and diseases will be foundational to efforts to predict possible interventions.
This text is an evolving and collaborative document that seeks to incorporate the ever-expanding body of information related to SARS-CoV-2 and COVID-19.

## Where to Contribute
Introduce Yourself (GitHub Issue) [https://github.com/greenelab/covid19-review/issues/17](https://github.com/greenelab/covid19-review/issues/17)

Community Chat (Gitter Room) [https://gitter.im/covid19-review/community](https://gitter.im/covid19-review/community)

More Info (GitHub Readme) [https://github.com/greenelab/covid19-review#sars-cov-2-and-covid-19-an-evolving-review-of-diagnostics-and-therapeutics](https://github.com/greenelab/covid19-review#sars-cov-2-and-covid-19-an-evolving-review-of-diagnostics-and-therapeutics)


## Introduction {.page_break_before} 

### General Background

On January 21, 2020, the World Health Organization (WHO) released its first report concerning what is now known as the Coronavirus disease 2019 (COVID-19) [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf].
This infectious disease came to international attention on December 31, 2019 following an announcement by national officials in China about 44 cases of a respiratory infection of unknown cause.
The first known cases were located in Wuhan City within the Hubei province of China, but the disease spread rapidly beyond Wuhan within China and subsequently around the world.
At the time of the first situation report [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf], 282 confirmed cases had been identified, primarily in China, but also 1-2 exported cases had been identified in several neighboring countries (Thailand, Japan, and the Republic of Korea).
One week later, 4593 confirmed cases had been identified, spanning not only Asia, but also Australia, North America, and Europe [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200128-sitrep-8-ncov-cleared.pdf].
On March 11, 2020, WHO formally classified the situation as a pandemic [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf].
By WHO Situation Report 61, released on March 20, 2020, 266,073 confirmed cases had been reported worldwide, with cases on every continent except Antarctica [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200321-sitrep-61-covid-19.pdf].
At this time, over 11,000 deaths had been reported worldwide.

[Note: Maybe add a graph here, update as new reports come out.]

COVID-19 is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).
SARS-CoV-2 is a coronavirus, a family of RNA viruses known to cause respiratory and intestinal infections in humans and other species.
Infectious diseases of global concern have previously been associated with coronaviruses, including Severe Acute Respiratory Syndrome (SARS-CoV) and Middle East respiratory syndrome (MERS-CoV) [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9]; however, neither of these reached pandemic status, owing to proper containment procedures (SARS) or intrinsic limitations in virus transmission (MERS).
Additionally, there are four endemic human coronaviruses that rarely progress beyond the mild symptoms associated with the common cold [@doi:10.1038/s41579-018-0118-9].
The precise identity of SARS-CoV-2 virus was unknown until approximately January 12, 2020, when Chinese officials released its genetic sequence to aid in worldwide efforts to diagnose the disease [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf].
As researchers worldwide work to characterize SARS-CoV-2 and COVID-19, information about the transmission and life cycle of the virus as well as the diagnosis and treatment of the disease is emerging rapidly.
In this review, we seek to consolidate information about the virus in the context of related viruses and to synthesize what is known about the diagnosis and treatment of COVID-19 and related diseases.
This is a real-time, collaborative effort that welcomes submissions from scientists worldwide.

### About Coronaviruses

#### Classification:
Coronaviridae, commonly called Coronavirus, is a family of positive-sense RNA viruses [@isbn:0781760607]. 
Members of this family include the Middle East respiratory syndrome-related coronavirus (MERS-CoV) and Severe acute respiratory syndrome coronavirus (SARS-CoV), which are two species, as well as SARS-CoV-2 [@url:https://en.wikipedia.org/wiki/Coronaviridae#Taxonomy].

#### Virion, morphology, structure:
Coronavirus virions are spherical with diameters ranging between 100 to 160nm. 
The virion has a lipid envelope with spike (S) glycoproteins on the surface creating a distinctive "crown" shape, for which the family of viruses was named [@isbn:0781760607]. 

#### Genome structure and replication:
Coronavirus genomes are single-stranded non-segmented positive RNA (ssRNA+) that are 27 to 32kb in length. 
The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp in length [@doi:10/ggjr43].
Coronavirus genomes are comprised of a replicase gene that is 2/3 of the genome and the remaining 1/3 that encodes structural proteins including: spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins. 
There are additional accessory genes that are present depending on the strain. 
The non-structural polypeptide is translated into 16 non-structural proteins (nsp1-16, except Gammacoronaviruses that do not contain nsp1), which form replication machinery to synthesize viral RNA [@doi:10.1002/jmv.25681]. 

Replication initiates with the binding of the virus to a host cell via the S protein. 
The viral membrane fuses with the endosomal membrane to release the viral genome into the host cytoplasm. 
The replicase gene is translated and assembled into the viral replicase complex, which can synthesize dsRNA genome from the genomic ssRNA(+).
The dsRNA genome is then transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes [@doi:10.1007/978-1-4939-2438-7_1; @url:https://viralzone.expasy.org/30?outline=all_by_species].  

#### Transmission:
In general, respiratory viruses like coronavirus can have multiple routes of person-to-person transmission including droplet transmission (i.e. inhalation for cough, sneeze), aerosol transmission (i.e. virus suspended in air), and contact transmission (i.e. contact with oral, nasal, and eye mucous membranes). 
While there does not appear to be experimental evidence to conclude which mode of transmission different coronaviruses use [@doi:10.1038/s41368-020-0075-9], it is suggested that the coronavirus person-to-person transmission is droplet-based using simulations [@doi:10.1056/NEJMoa032867] and trace studies[@doi:10.3390/ijerph15122757].

### Origin of SARS-CoV-2 
[MERS and SARS? Where to put this?]
[Specifically about SARS-CoV-2/COVID19?]
The origin of the SARS-CoV-19 virus is not yet fully understood.
Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-19 is unlikely to have originated from a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
Among known coronaviruses, SARS-CoV-19 has the closest overall sequence similarity to RaTG13 (~96%) found in a *Rhinolophus affinis* bat [@doi:10.1038/s41586-020-2012-7], while the receptor binding domain (RBD) is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253].
This suggests that SARS-CoV-19 may have originated in viral reservoirs of similar hosts, however current evidence cannot discriminate an origin of the virus before or after zoonotic transfer to humans [@doi:10.1038/s41591-020-0820-9].

### Disease (COVID19)

#### Mechanisms of Coronavirus-driven Disease in Humans

Coronaviruses are known to cause respiratory illnesses in humans through the following possible mechanisms...[Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important]

#### Symptoms

*Presentation of COVID-19:* 
Information is rapidly becoming available about the wide range of symptoms that can be associated with COVID-19 as well as the range of symptom severity, onset from exposure, and possible risk or protective factors...

#### Vaccines for Viruses: Strategies for and challenges to development

What information is needed to develop a vaccine? How have vaccines for other viruses such as H1N1 been developed?

#### Diagnostics and Therapeutics for Viruses

Two major concerns within diagnosis include the detection of current infections in individuals with and without symptoms, and the detection of past exposure without an active infection.
In the latter category, identifying whether individuals can develop or have developed sustained immunity is also a major consideration.

Within therapeutics, some possible efforts include efforts to identify strategies for the management of symptoms as well as the development of antivirals...

In this review, we seek to consolidate information about efforts to develop strategies for diagnosis and therapeutics as new information is released by the scientific community.


## Pathogenesis

### Mechanism of Host Infection by SARS-CoV-2

This section would also be great for the introduction of zoonotic diseases which has been shown to be the origin of SARS-CoV2.

#### Primary Transmission and Viral Entry

[How does SARS-CoV-2 enter human cells?]
[What cells are primary infection sites for SARS-CoV-2?]
[What structural aspects allow for viral entry]?

#### Viral Replication and Spreading

[Renamed section with transmission dynamics below] 
[Basic introduction into replication cycle]
[What are the routes of transmission]

#### Reproduction Number and Dynamics of Transmission

Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
The basic reproduction number, R~0~, is the expected number of new infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
The effective reproduction number, R~t~, describes how the reproduction number may change over time, and is used to quantify deviations in R from R~0~, for example as some fraction of the population becomes infected, or as interventions are put into place.
R~0~ and R~t~ can be estimated directly from epidemiological data or inferred using mathematical modeling.
Modeling approaches are typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions [@doi:10.1126/science.272.5258.74]. 

R~0~ for COVID-19 is estimated to lie in the range R~0~=1.4-6.5 [@doi:10/ggjvr7; @doi:10.1093/jtm/taaa021; @doi:10.1126/science.abb3221]; estimates vary considerably depending on the data and the methods used.
Most estimates currently derive from populations in Asia, since outbreaks in Europe and North America are more recent.
Data-derived estimates (i.e. those that do not incorporate SIR-type models into their analysis) typically predict lower values of R~0~.
For data-derived estimates, in one study of international cases, the predicted value is R~0~=1.7 [@doi:10.1101/2020.03.21.20040329], in China (both Hubei province and nationwide), the value is predicted to lie in the range R~0~=2.0-3.6 [@doi:10.2139/ssrn.3524675; @doi:10.1101/2020.01.25.919787; @doi:10/ggjvr7], and on a cruise ship where an outbreak occurred, predicted R~0~=2.28 [@doi:10.1016/j.ijid.2020.02.033].
SIR model-derived estimates of R~0~ range from 2.0 - 6.5 in China [@doi:10.3390/jcm9020462; @url:https://doi.org/10.1101/2020.01.27.20018952; @doi:10.1101/2020.01.23.916726; @doi:10.1101/2020.01.23.20018549] to R~0~=4.8 in France [@doi:10.1101/2020.03.22.20040915]. 
Using the same model as for the French population, this study estimated R~0~=2.6 in South Korea [@doi:10.1101/2020.03.22.20040915], which is consistent with other studies [@tag:Park2020_distancing].
From a meta-analysis of studies estimating R~0~, [@doi:10.1093/jtm/taaa021] predict the median as R~0~=2.79. 

Inference of the effective reproduction number can provide insight into how populations respond to an infection, and the effectiveness of interventions.
In China, R~t~ was predicted to lie in the range 1.6-2.6 in Jan 2020, before travel restrictions [@doi:10/ggptcf].
R~t~ decreased from 2.35 one week before travel restrictions were imposed (Jan 23, 2020), to 1.05 one week after.
Using their model, the authors also estimate the probability of new outbreaks occurring: the probability of a single individual exporting virus causing a large outbreak is 17-25% assuming MERS-like or SARS-like transmission, and the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%.
An independent study came to similar conclusions: in a two-week period before Jan 23 finding R~t~=2.38, and decreasing to R~t~ = 1.34 (using data from Jan 24 to Feb 3) or R~t~=0.98 (using data from Jan 24 to Feb 8) [@doi:10.1126/science.abb3221].
In South Korea, R~t~ was inferred for Feb-Mar 2020 in two cities: Daegu (the center of the outbreak), and Seoul [@tag:Park2020_distancing].
Metro data was also analyzed to estimate the effects of social distancing measures.
R~t~ decreased in Daegu from around 3 to <1 over the period that social distancing measures were introduced.
In Seoul, R~t~ decreased slightly, but remained close to 1 (and larger than R~t~ in Daegu).
This highlights that social distancing measures appeared to work to contain the infection in Daegu, but that in Seoul, R~t~ remains above 1, thus secondary outbreaks are possible.
It also shows the importance of region-specific analysis: the large decline in case load nationwide is mainly due to the Daegu region, and could hide persistence of the epidemic in other regions, such as Seoul and Gyeonggi-do. 
Similarly in Iran, estimates of R~t~ declined from 4.86 in the first week to 2.1 by the fourth week after the first cases in Iran were reported [@doi:10.1101/2020.03.20.20038422]. The authors attributed this decline to the effects of self-quarantine and government reductions in working time.

### Immune Response to SARS-CoV-2

[Cellular responses to SARS-CoV-2 infection]
[What is causing neutropenia and lymphopenia observed in COVID-19 patients]
[Antibody production against SARS-CoV-2 by patient who recovered vs patient who did not recover]
[Cytokines and other soluble factors contribution to immune response]

### Systems level approaches for understanding SARS-CoV-2 pathogenesis

Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].

The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet.
A significant breakthrough in SARS-CoV-2 research was achieved through the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7].
Multiple research groups have drafted the genome sequence of SARS-CoV-2 based on sequencing of clinical samples collected from bronchoalveolar lavage fluid (BALF) [@doi:10.1038/s41586-020-2008-3; @doi:10.1038/s41586-020-2012-7] or from BALF, throat swabs, or isolates of the virus cultured from BALF [@doi:10/ggjr43].
Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@doi:10/ggjr43].
However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21), indicating a possible origin in bats [@doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7].

The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection.
Availability of the genome sequence of SARS-CoV-2 enhances the potential for subsequent proteome-level studies to provide further mechanistic insights into the virus' complex pathogenesis.
Of note, the cryo-electron microscopy structure of the SARS-CoV-2 spike (S) glycoprotein, which plays an important role in the early steps of viral infection, was reported very recently [@doi:10.1126/science.abb2507].
Even though no comprehensive proteomic analysis of the pathogen or of patients suffering from its infection has yet been reported, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Caco–2 cells as an infection model [@doi:10.21203/rs.3.rs-17218/v1].
The authors observed SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.

There is a high level of sequence homology between SARS-CoV-2 and SARS-CoV, and sera from convalescent SARS-CoV patients might show some efficacy to cross-neutralize SARS-CoV-2-S-driven entry [@doi:10.1016/j.cell.2020.02.052].
However, despite the high level of sequence homology, certain protein structures might be immunologically distinct, prohibiting effective cross-neutralization across different SARS strains [@doi:10.1101/2020.03.21.990770].
Consequently, earlier proteome-level studies on SARS-CoV can also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074].
Considering the paucity of omics-level big data sets for SARS-CoV-2 up until now, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for computational and bioinformatics research on SARS-CoV-2.


## Diagnostics

### Current Strategies for Diagnosing COVID-19 and Similar Viral Infections

Given the heterogeneity of symptom presentation across patients with COVID-19, the development of standardized protocols for testing samples for SARS-CoV-2 is urgent.
Following the release of the genetic sequence of the virus by Chinese officials on January 12, 2020, the first tests for detecting the virus were released on XX, 2020.
These tests used the following approach to identify the active virus in patient samples...
However, many countries have struggled to acquire the tests required to keep pace with the epidemic.
[Why is it so difficult to scale up testing? What are some of the considerations?]

### Possible Alternatives to Current Practices for Identifying Active Cases

[Are there other approaches that have worked for diagnosing other viruses at a rapid pace in large numbers of people?]
[What are some approaches people are currently testing for detecting live viruses, especially SARS-CoV-2?]

### Detection of Past Exposure and/or Sustained Immunity

[What are approaches that allow us to detect past exposure for other viruses?]
[What efforts are underway to develop similar approaches for SARS-CoV-2?]
[What is sustained immunity and what are the indicators?]

### Limitations to Implementation of Large-Scale Testing

[Right now, reagent supply is an issue. Are there others concerns that are likely to emerge?]

### Strategies and Considerations for Determining Whom to Test

[If it's not possible to test everyone, what strategies exist for selecting who to test?]
[Are these strategies likely to change over time? Presumably there are different stages of managing spread vs mitigating severity once it's already at high prevalence?]


## Therapeutics

Given the rapid predicted spread of the disease, the development of therapeutics will be critical to mitigating its effect on health and the mortality rate.
Typically, therapeutics can take a few forms.
First, the treatment and reduction of symptoms can result in the reduction of the severity and risk associated with an active infection.
Second, the development of antiviral drugs can drive a reduced recovery time for patients by inhibiting the development of the virus once an individual is infected.
Finally, vaccines present a strategy for bolstering the immune response of the populus broadly to the virus, resulting in a lower rate of infection.
All three of these strategies have been valuable elements of responses to other viruses, including coronaviruses, and are being investigated by researchers at present.
Additionally, there have been suggestions within the scientific community that nutraceutical or dietary supplement interventions may prime an individual’s immune system to prevent or lessen the impact of RNA virus infections [@doi:10.1016/j.pcad.2020.02.007; @doi:10/ggkd3b].
In the following sections, we critically appraise the literature surrounding the repurposing of existing treatments and development of novel therapeutics for the prevention, mitigation, and treatment of coronavirus infections.  

### Treatment of Symptoms

The clinical picture of SARS-CoV-2 infection differs dramatically between individuals.
Some are asymptomatic.
Others experience mild COVID-19 symptoms, such as cough, sore throat and fever, while the most severe cases of COVID-19 include severe complications including pneumonia and Acute Respiratory Distress Syndrome (ARDS), which can lead to respiratory failure and death [@url:https://www.cdc.gov/coronavirus/2019-ncov/hcp/clinical-guidance-management-patients.html].
Vaccines are one avenue to mitigate harm from viral pathogens, but in the case of a rapidly growing pandemic the longer timeframe of vaccine development and distribution means that there can be a key for treatments that palliate symptoms to avoid the most severe outcomes from infection.

#### Tocilizumab

A recent study carried out on a sample of 191 adult COVID-19 in-patients at two Wuhan hospitals found that blood samples taken at admission contained significantly higher concentrations of interleukin-6 (IL-6) in patients who ultimately deceased compared to those who survived; average concentrations of IL-6 remained higher in the deceased group than the surviving group throughout hospitalization [@doi:10/ggnxb3].
This suggests that these individuals may be experiencing a "cytokine storm", which refers to an excessive inflammatory response.
IL-6 plays a key role in this response [@doi:10.1128/mmbr.05015-11].
IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, which are immune system regulators that are primarily responsible for immune cell differentiation.
Specifically, IL-6 promotes the differentiation of activated B cells into immunoglobulin-producing plasma cells [@pmid:9052836] and acts as a growth factor for hybridoma and myeloma cells [@pmid:8431556; @pmid:2104241].
In addition, IL-6 also induces the differentiation of naïve CD4+ T cells into effector T-cell subsets [@doi:10.1158/2326-6066.CIR-14-0022].
In this way interleukins regulate both the pro- and anti-inflammatory responses.
In this context, the observation of elevated IL-6 in patients who died may reflect an over-production of proinflammatory interleukins.

In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system, ensuring homeostasis.
The introduction of the S-protein from SARS-CoV to mouse macrophages was found to increase production of IL-6 and TNF-α [@doi:10.1016/j.virusres.2007.02.007], and deceased SARS-CoV patients were found to have intermediate levels of IL-6, IL-1𝛽, and TNF-α expressed in a number of ACE2-expressing cell types sampled from the lung and bronchial tissues during autopsy [@doi:10.1002/path.2067].
However, other reports found the severe respiratory condition ARDS to be associated with elevated concentrations of IL-6 in BALF, but that concentrations of Tumor Necrosis Factor α (TNF-α) and IL-1𝛽 decreased with the onset of ARDS [@doi:10.1164/ajrccm.164.10.2104013].
These cytokines enhance the pro-inflammatory reaction by increase acute-phase signaling, trafficking of immune cells to the site of primary infection, epithelial cell activation, and secondary cytokine production.
The acute phase response to infection results in the heavily damage of the endotelium of blood vessels, which disrupts the balance between pro and anti-inflammatory response [@doi:10.1164/ajrccm.164.10.2104013].
Thus, the holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death as per Acute Distress Respiratory Syndrome (ADRS) or Severe Acute Respiratory Syndrome (SARS) [@doi:10.1007/s00281-017-0629-x], also caused by the new coronavirus.
Recently Chinese and Italian doctors have found that the the Tocilizumab (actemra, by Roche), a drug commonly used in the rheumatoid arthritis, may palliate the most severe symptoms associated with COVID-19.

##### Anticipated Mechanism

Human IL-6 is a glycoprotein of 26 kDa and it consists of 184 amino acids containing 2 potential N-glycosylation sites and four cysteine residues. IL-6 binds to its receptor either in the insoluble (IL-6R) and soluble (sIL-6R) form.
The receptor specificity determines the type of signaling. Specifically, the binding of IL-6 to the cell membrane receptor IL-6R gives rise to the "classical transduction of the signaling", while to binding to sIL-6R generate the so called "trans-signaling" [@doi:10.1016/j.cytogfr.2012.04.001; @doi:10.1042/bj3000281].
IL-6 signaling occurs through 3 independent pathways: the Janus-activated kinase (JAK)-STAT3 pathway, the Ras/Mitogen-Activated Protein Kinases (MAPK) pathway and the Phosphoinositol-3 Kinase (PI3K)/Akt pathway [@doi:10.3389/fimmu.2016.00604].
The ultimate result of the IL-6 cascade is to direct transcriptional activity of various promoters of pro-inflammatory cytokines, such as IL-1 and TFN, including IL-6 own regulation through the activity of NF-κB [@doi:10.3389/fimmu.2016.00604].
Particularly, IL-6 synthesis is tightly regulated both transcriptionally and post-transcriptionally . In this context, it has been shown that viral proteins can enhance transcription of the IL-6 gene, via strengthening the DNA-binding activity between several transciptional factors and IL-6 gene-cis-regulatory elements [@doi:10.2492/inflammregen.33.054].
Tocilizumab is a humanised monoclonal antibody that binds both to the insoluble and soluble receptor of IL-6, de facto inhibiting the IL-6 immune cascade.

##### Current Evidence

Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear sings of improvements, according to their results [@url:https://www.genengnews.com/virology/coronavirus/genentech-launches-phase-iii-trial-of-actemra-as-coronavirus-treatment/].
Also, The AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection [@url:https://www.univadis.co.uk/viewarticle/covid-19-italy-launches-an-independent-trial-on-tocilizumab-715741].
Together with these independent trials, Roche, also in collaboration with the FDA, will start a randomised, double-blind, placebo-controlled phase III trial early April.
The trial will enroll 330 patients globally, which will be followed for 60 days upon use of the drug via injection to analyse its efficiency/safety (Biopharma-reporter.com).
However, previous studies on RA showed that in patients treated with TCZ the rate of incident infections in clinical practice patients was higher than the one observed during clinical trial [@doi:10.1093/rheumatology/ker223].
Also, RA patients with chronic hepatitis B (HB) infection showed high risk of HB virus reactivation upon TCZ administration in combination with other RA drugs [@doi:10.1111/1756-185X.13010].
These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated by the cytokine storm, and preventing the worsening of the clinical picture of the patients by potential new viral infections.
This aspect is probably crucial to be investigated further in the trials that are about to start.
Perhaps, the TCZ treatment would best suit patients with severely compromised lungs due to the Covid-19 infection and are therefore at greater risk of death, in order to stop the uncontrolled immune response before it’s too late.

##### Summary

Summarize the state of the symptom management approach.

### Small Molecule Drugs for Targeting SARS-CoV-2

The replication cycle of a virus within an epithelial host cell includes 6 basic steps which can be shortly summarised as follow: i) attachment of the virus to the host cell; ii) penetration by endocytosis; iii) uncoating, classically defined as the release of viral contents into the host cell; iv) biosynthesis, during which the viral genetic material enters the nucleus where it gets replicated; v) assembly, where viral proteins are translated and new viral particles are assembled; vi) release, when the new viruses are released into the extracellular environment [@isbn:978-1405136457].
Antiviral drugs do not kill the virus, rather they inhibit its amplification by impairing one of these steps.
Nowadays, many of these drugs act during the biosynthesis step in order to inhibit the virus’ genetic material replication.
Importantly, SARS-CoV-2 is an RNA virus.
Differently from a DNA virus, which can use the host enzymes to propagate itself, RNA viruses depends on their own polymerase, the RNA-dependent RNA polymerase (RdRP), in order to be replicated [@doi:10.1007/978-1-4939-2438-7; @doi:10.1002/jmv.25761].

#### Nucleotide Analogs

Why one might use nucleotide analogs.

##### Avigan

Avigan (Flavipiravir, from Toyama chemical Fujifilm) is a drug which has been found effective to block viral amplification in the Influenza virus infection.

###### Anticipated Mechanism

Specifically, Avigan is a nucleoside precursor efficiently recognised as guanosine and adenosine analogue by the virus polymerase (RNA-dependent RNA polymerase).
While a single incorporation does not influence RNA transcription, multiple events of incorporation lead to the arrest of RNA synthesis [@doi:10.1371/journal.pone.0068347].
Importantly It was already shown that Avigan is able to give 100% coverage against the Ebola virus in mice [@doi:10.1016/j.antiviral.2014.02.014].
Furthermore, there are evidences that drug might also work against Corona virus infection.
For instance, a recent study showed its effectiveness compared to other antivirals such as lopinavir and ritonavir [@doi:10.1016/j.eng.2020.03.007].
The drug was tested on a sample of 80 patients (35 experimental sample, 45 control group) and increased the speed of recovery (measured as viral clearance from the patient by RT-PCR) of about 4 days, compared to the control sample treated with the other anti-virals, such as Lopinavir and Ritonavir.
Also lung body scan seem to look better in about 91% of the patients analysed [@doi:10.1016/j.eng.2020.03.007].
However, the size of the sample is too small to give a powerful statistics, as well as the choice of the patients did not take into consideration important factors such as previous clinical conditions, sex, while there was no age categorisation.

###### Current Evidence

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

###### Summary

Summarize the state of the antiviral approach.

##### Remdesivir

Remdesivir (GS-5734) was developed by Gilead Sciences to treat Ebola.
It does not have any FDA-approved use.
Although a clinical trial in the Democratic Republic of Congo found some evidence of effectiveness against ebola, two antibody preparations were found to be more effective, and Remdesivir was not pursued [@doi:10.1056/NEJMoa1910993].

###### Anticipated Mechanism

Remdesivir is metabolized to GS-441524, an adenosine analog that inhibits a broad range of polymerases and then evades exonuclease repair causing chain termination [@doi:10.1074/jbc.AC120.013056; @doi:10.1128/mBio.00221-18; @doi:10.1038/s41422-020-0282-0].
Although it was developed against Ebola, it also inhibits the MERS-CoV and SARS-CoV polymerase and inhibits coronavirus replication in cell culture assays with submicromolar IC50s [@doi:10.1126/scitranslmed.aal3653].
It also inhibits SARS-CoV-2, showing synergy with chloroquine [@doi:10.1038/s41422-020-0282-0].

###### Current Evidence

In addition to the previous work showing Remdesivir to be an effective treatment for viral pathogens such as SARS-CoV and MERS-CoV in cultured cells and animal models, a recent study found that administration of Remdesivir to non-human primate models resulted in 100% protection against infection by the Ebola virus.
Remdesivir has also been reported to inhibit SARS-CoV-2 infection in a human cell line sensitive to the virus [@doi:10.1038/s41422-020-0282-0].

The effectiveness of Remdesivir for treating patients with COVID-19 is currently under investigation.
Remdesivir has been used on some COVID-19 patients under compassionate use guidelines [@doi:10.1126/science.abb7243, @doi:10.1056/NEJMoa2001191; @doi:10.1101/2020.03.09.20032896].
All were in late stages of COVID-19 infection, and these reports are inconclusive about the drug's efficacy.
Remdesivir recently entered controlled clinical trials, and as of March 2020, there are six clinical trials underway to treat COVID-19 patients at both early and late stages of infection and in combinations with other drugs [@doi:10.1038/s41422-020-0282-0; @url:https://clinicaltrials.gov/ct2/show/NCT04292730; @url:https://clinicaltrials.gov/ct2/show/NCT04292899; @url:https://clinicaltrials.gov/ct2/show/NCT04280705; @url:https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR; @url:https://clinicaltrials.gov/ct2/show/NCT04252664; @url:https://clinicaltrials.gov/ct2/show/NCT04257656].

###### Summary

Remdesivir is a major drug candidate since it attacks the virus with high potency and known mechanism.
Moreover, one of the most successful therapies for viral diseases is to target the viral replication machinery, which are typically virally encoded polymerases.
Small molecule drugs targeting viral polymerases are the backbones of treatments for other viral diseases including HIV and Herpes.
Note that the HIV and Herpes polymerases are a reverse transcriptase and a DNA polymerase respectively, whereas SARS-CoV-2 encodes an RNA dependent RNA polymerase, so most of the commonly used polymerase inhibitors are not likely to be active against SARS-CoV-2.
In clinical use, polymerase inhibitors show short term benefits for HIV patients but for long term benefits they must be part of combination regimens.
They are typically combined with protease inhibitors, integrase inhibitors and even other polymerase inhibitors.   

###### Current Evidence

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

###### Summary

Summarize the state of the antiviral approach.

#### Protease Inhibitors

Why it may be useful

##### Protease Inhibitor 1

###### Anticipated Mechanism

###### Current Evidence

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

###### Summary

Summarize the state of the antiviral approach.

#### Molecules Targeting the Viral Envelope

Why it may be useful

##### Viral Envelope Targeter 1

###### Anticipated Mechanism

###### Current Evidence

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

###### Summary

Summarize the state of the antiviral approach.

### Drugs Targeting Host Proteins

Brief background on the therapeutic.

#### Drug (or drug class) 1

##### Viral Entry Receptors

Why it may be useful

###### Current Evidence

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

###### Summary

Summarize the state of the antiviral approach.

### Broad-Spectrum Pharmaceuticals

#### Hydroxychloroquine

#### Nutraceuticals

Considering the current pandemic, scientists and the medical community are scrambling to repurpose or discover novel host-directed therapies for which nutraceuticals hold some promise.
Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value designed for the prophylaxis and treatment of disease [@doi:10.1080/10408390902841529; @doi:10.1038/nrcardio.2016.103].
Nutraceuticals purported to boost the immune response, reduce  immunopathology, exhibit antiviral activities or prevent acute respiratory distress syndrome (ARDS) are being considered for their potential therapeutic value [@doi:10/ggkd3b].
A host of potential candidates have been highlighted in the literature that target various aspects of the COVID-19 viral pathology, while others are thought to prime the host immune system.
These candidates include vitamins and minerals along with extracts and omega-3 polyunsaturated fatty acids (n-3 PUFA) [@doi:10.1002/jmv.25707].
Considerable evidence in vitro and in vivo suggests that nutraceuticals containing phycocyanobilin, N-acetylcysteine, glucosamine, selenium or phase 2 inductive nutraceuticals (e.g. ferulic acid, lipoic acid, or sulforaphane) can prevent or modulate RNA virus infections via amplification of the signaling activity of mitochondrial antiviral-signaling protein (MAVS) and activation of toll-like receptor 7 (TLR7) [@doi:10.1016/j.pcad.2020.02.007]. 
While promising, further animal and human studies are required to assess the therapeutic potential of these various nutraceuticals against COVID-19. 

##### n-3 PUFA

Another potential nutraceutical that has exhibited beneficial effects against various viral infections is n-3 PUFA [@doi:10.1002/jmv.25707], such as eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA).
EPA and DHA intake can come from a diet high in fish intake or through dietary supplementation with fish oils or purified oils [@doi:10.3390/molecules22111964].


###### Potential Mechanisms

N-3 PUFA nutraceuticals can mediate inflammation and they have the capacity to modulate the adaptive immune response [@doi:10.1111/j.1365-2125.2012.04374.x; @doi:10.3390/molecules22111964; @doi:10.1038/nrcardio.2016.103].
Another potential mechanism by which n-3 PUFA could exert beneficial effects against viral infections is by acting as precursor molecules for the biosynthesis of endogenous specialised proresolving mediators (SPM) like protectins and resolvins that actively resolve inflammation and infection [@doi:10.1016/j.immuni.2014.02.009]. 

###### Current Evidence

SPM have exhibited beneficial effects against a variety of lung infections including RNA viruses [@doi:10.1038/nri.2015.4].
Indeed, protectin D1 has been shown to increase survival from H1N1 viral infection in mice by affecting the viral replication machinery [@doi:10.1016/j.cell.2013.02.027].
Moreover, not all studies are in agreement that n-3 PUFA are effective against infections [@doi:10.3945/jn.109.108027]. 
Indeed, the effectiveness of n-3 PUFA against infections is dependent on the dosage, timing, and the specific pathogens responsible [@doi:10.1016/j.jinf.2016.10.001].  

###### Summary

However, the overall lack of human studies in this area means there is limited evidence as to whether these nutraceuticals could affect COVID-19 infection.  

##### Zinc Supplements

There is evidence that nutrient supplements may exhibit some benefit against RNA viral infections.
Zinc is a trace metal obtained from dietary sources or supplementation that is important for the maintenance of immune cells involved in adaptive and innate immunity [@doi:10.1016/j.abb.2016.03.022].
Zinc supplements can be administered orally as a tablet or as a lozenge and they are available in many forms such zinc picolinate, zinc acetate, and zinc citrate.
Zinc is also available from dietary sources including meat, seafood, nuts, seeds, legumes, and dairy.

###### Potential Mechanisms

The role of zinc in immune function has been extensively reviewed [@doi:10.1016/j.abb.2016.03.022].
Zinc is an important signaling molecule and zinc levels can alter host defense systems. 
In inflammatory situations such as an infection, zinc can regulate leukocyte immune responses and it can activate the nuclear factor kappa-light-chain-enhancer of activated B cells (NF-kB), thus altering cytokine production [@doi:10.4049/jimmunol.179.6.4180; @doi:10.1067/mlc.2001.118108]. 
In particular, zinc supplementation can increase natural killer cell levels, which are important cells for host defense against viral infections [@doi:10.1126/science.1198687; @doi:10.1016/j.abb.2016.03.022].

###### Current Evidence

Adequate zinc intake has been associated with reduced incidence of infection [@doi:10.1093/ajcn/85.3.837] and antiviral immunity [@doi:10.1093/advances/nmz013].
Similarly, a randomized, double-blind, placebo-controlled trial that administered zinc supplementation to elderly subjects over the course of a year found zinc deficiency to be associated with increased susceptibility to infection and that zinc deficiency could be prevented through supplementation [@doi:10.1093/ajcn/85.3.837].
Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms [@doi:10.1331/1544-3191.44.5.594.Hulisz; @doi:10.2174/1874306401105010051].
In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a flourescently tagged SARS-CoV) and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015].

###### Summary

While overall there is encouraging data for zinc supplementation against the common cold and viral infections, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19, thus further research is imperative.
However, it would be advisable to maintain a healthy diet to ensure an adequate zinc status to prevent the likelihood of an infection. 

##### Vitamin C

Vitamins B, C, D, and E have also been suggested as potential nutrient supplement interventions for COVID-19 [@doi:10.1002/jmv.25707; @doi:10.20944/preprints202003.0347.v1].
In particular vitamin C has been proposed as a potential prophylactic and therapeutic agent against COVID-19. 
Vitamin C can be obtained via dietary sources such as fruit and vegetable or via supplementation.  

###### Potential Mechanisms
Vitamin C plays a signifiant role in promoting immune function due to its effects on various immune cells. 
Vitamin C affects inflammation by modulating cytokine production, decreasing histamine levels, enhancing the differentiation and proliferation of T- and B-lymphocytes, increasing antibody levels, and protecting against the negative effects of reactive oxygen species amongst other effects [@doi:10.1155/2014/426740; @doi:10.1007/s00210-013-0880-1; @doi:10.3390/nu9111211]. 
During viral infections vitamin C is utilised as evinced by lower concentrations in leukocytes and lower concentrations of urinary vitamin C. 
Post-infection, these levels return to baseline ranges [@doi:10.1177/003693307301800102; @doi:10.1111/j.1749-6632.1975.tb29312.x; @doi:10/fd22sv; @doi:10.3390/nu9040339; @doi:10.1079/bjn19920004]. 

###### Current Evidence

A recent meta-analysis found consistent support for regular vitamin C supplementation reducing the duration of the common cold, but that supplementation with vitamin C (> 200 mg) failed to reduce the incidence of colds [@doi:10.1002/14651858.CD000980.pub4].
Individual studies have found Vitamin C to reduce the susceptibility of patients to lower respiratory tract infections such as pneumonia [@doi:10.1097/00006454-199709000-00003].
A recent meta-analysis has demonstrated in twelve trials that vitamin C supplementation reduced the length of stay of patients in intensive care units (ICUs) by 7.8% (95% CI: 4.2% to 11.2%; p = 0.00003).
Furthermore, high doses (1-3 g/day) significantly reduced the length of an ICU stay by 8.6% in six trials (p = 0.003).
Vitamin C also shortened the duration of mechanical ventilation by 18.2% in three trials in which patients required intervention for over 24 hours (95% CI 7.7% to 27%; p = 0.001)  [@doi:10.3390/nu11040708]. 
Despite these findings, the CITRUS ALI study failed to show a benefit of a 96-hour infusion of vitamin C to treat ARDS, which is a severe complication of COVID-19 infection [@doi:10.1001/jama.2019.11825].

###### Summary

The evidence suggests that vitamin C supplementation can shorten the duration of a cold, reduce an individual's susceptibility to infections, and shorten a patient's stay in ICU when administered at high doses.
While there is anecdotal evidence from China and the United States that vitamin C is currently being used for the treatment of COVID-19,  further controlled research is required.

##### Vitamin D

In terms of other dietary supplements, vitamin D can modulate the adaptive and innate immune system and has been associated with various aspects of health. 
Vitamin D can be sourced through diet or supplementation, but it is mainly biosyntheised by the body on exposure to sunlight.

###### Potential Mechanisms

Vitamin D deficiency is associated with an increased susceptibility to infection [@doi:10.2310/JIM.0b013e31821b8755].
In particular, vitamin D deficient patients are at risk of developing acute respiratory infections [@doi:10.1016/j.jsbmb.2012.11.017] and ARDS [@doi:10.1016/j.jsbmb.2012.11.017]. 
1,25-dihydroxyvitamin D3 is the active form of vitamin D that is involved in adaptive and innate resopnses, whereby the vitamin D receptor is expressed in various immunne cells and vitamin D is an immunomodulator of antigen presenting cells, dendritic cells, macrophages, monocytes, and T- and B-lymphocytes [@doi:10.2310/JIM.0b013e31821b8755; @doi:10.1016/j.coph.2010.04.001].
Due to its potential immunomodulating properties, vitamin D supplementation may be advantageous to maintain a healthy immune system. 

###### Current Evidence

A recent preprint postulated that an individual’s vitamin D status may significantly affect one’s risk of developing COVID-19. 
This hypothesis was derived from the fact that the current pandemic occurred in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer, the number of cases was low at that time coinciding with higher 25-hydroxyvitamin D concentrations [@doi:10.20944/preprints202003.0235.v2].
The authors suggest that people at risk of developing COVID-19 should increase their vitamin D3 intake to reach 25-hydroxyvitamin D plasma concentrations above 40–60 ng/ml.
The authors also suggest supplementation of vitamin D to treat infected patients and to prevent infection in hospital staff.
While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels effect COVID-19 specifically. 
Likewise, it is assumed that COVID-19 may be seasonal, but there are multiple other factors at play that can affect vitamin D levels that need to be considered.
These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others [@doi:10.3390/nu7075111].   

###### Summary
  
Supplementation of vitamin D and maintaining a healthy diet for optimum vitamin D status warrants further investigation. 
This is particularly important considering ‘stay in place’ guidance has been implemented in many densely populated cities around the world.
This measure is likely to limit people’s exposure to sunlight and thus reduce endogenous synthesis of vitamin D, potentially weakening the immune system and increasing the risk of COVID-19 infection. 

##### Nutraceutical Conclusions

Despite all the potential benefits of nutraceutical and dietary supplement interventions presented, currently there is a paucity of clinical evidence to support their use for the prevention or mitigation of COVID-19 infections. 
Nevertheless, optimal nutritional status will undoubtedly prime an individual’s immune system to protect against the effects of acute respiratory viral infections by supporting normal maintenance of the immune system [@url:https://www.preprints.org/manuscript/202003.0199/v1].
Overall, supplementation of vitamin C, vitamin D, and zinc may be an effective method of ensuring their adequate intake to maintain optimal immune function, which may also convey beneficial effects against viral infections due to their immunomodulatory effects.
However, many supplements and nutraceuticals designed for various ailments are available in the United States and beyond that are not strictly regulated [@doi:10.1080/10408398.2019.1592107]. 
Indeed, there can be safety and efficacy concerns associated with many of these products.
Often, the vulnerable memeber of society can be exploited in this regard and unfortunately the COVID-19 pandemic is no different. 
The Food and Drug Administration (FDA) has issued warnings to several companies for advertising falsified claims in relation to the preventative and therapeutic capabilities of their products against COVID-19 [@url:https://www.fda.gov/news-events/press-announcements/coronavirus-update-fda-and-ftc-warn-seven-companies-selling-fraudulent-products-claim-treat-or].  
In light of these serious occurrences, it is pertinent to clarify that the nutraceuticals discussed in this review have been selected because of their possible relevance to the biological mechanisms that can beneficially affect viral and respiratory infections.
Therefore, further intensive investigation is required to establish the effects of these nutraceticals, if any, against COVID-19.  

### Biological Drugs for COVID-19

#### Neutralizing Antibodies

Monoclonal antibodies (mAbs) have revolutionized the way we treat human diseases. 
As a result, they have become some of the best-selling drugs in the pharmaceutical market in recent years [@doi:10.1186/s12929-019-0592-z].
There are currently 79 FDA approved mAbs on the market including antibodies for viral infections (e.g. Ibalizumab for HIV and Palivizumab for RSV) [@doi:10.1186/s12929-019-0592-z; @doi:10.1146/annurev-immunol-032712-095916].
Although vaccines remain the most important way to prevent viral infections, their development process is long and they fail to provide immediate prophylactic protection or treat ongoing infections [@doi:10.3390/v10120680].
For that reason, neutralizing antibodies have emerged to address these shortcomings.
Virus-specific neutralizing antibodies commonly target viral surface glycoproteins or host structures, thereby inhibiting viral entry through receptor binding intereference [@doi:10.3389/fmicb.2017.02323; @doi:10.1080/21645515.2017.1337614].
This section discusses current efforts in developing neutralizing antibodies against SARS-CoV-2 and how expertise gained from previous approaches for MERS-CoV and SARS-CoV may benefit antibody development.

##### Spike (S) Neutralizing Antibody

During the first SARS epidemic in 2002, nAbs were found in SARS-CoV infected patients [@doi:10.1111/j.1469-0691.2004.01009.x;@doi:10.1086/423286].
Several studies following up on these findings identified various S glycoprotein epitopes as the major targets of neutralizing antibodies against SARS-CoV [@doi:10.1517/14712590902763755].
The passive transfer of immune serum containing nAbs from SARS-CoV-infected mice resulted in protection of naïve mice from viral lower respiratory tract infection upon intranasal challenge [@doi:10.1128/jvi.78.7.3572-3577.2004].
Similarly, a meta-analysis suggested that administration of plasma from recovered SARS-CoV patients reduced mortality upon SARS-CoV infection [@doi:10.1093/infdis/jiu396]. 

Similar results have been observed for MERS-CoV infections, which emerged as the second coronavirus-related epidemic.
Neutralizing antibodies have been identified against various epitopes of the RBD of the S glycoprotein [@doi:10.1073/pnas.1402074111; doi:10.1128/JVI.00912-14].

###### Spike (S) Neutralizing Antibody Anticipated Mechanisms

Coronaviruses use trimeric spike (S) glycoproteins on their surface to bind to host cell receptors, such as ACE2, allowing for cell entry [@doi:10.1016/j.cell.2020.02.052; doi:10.1016/j.cell.2020.02.058].
Each S glycoprotein protomer is comprised of an S1 domain, also called the receptor binding domain (RBD), and an S2 domain.
The S1 domain binds to host cell receptors while the S2 domain facilitates the fusion between the viral envelope and host cell membranes [@doi:10.1517/14712590902763755].
Although targeting of the host cell receptor ACE2 shows efficacy in inhibiting SARS-CoV-2 infection [@doi:10.1038/nature02145], given the physiological relevance of ACE2 [@doi:10/bsbp49], it would be favorable to target virus-specific structures rather than host receptors.
This forms the rationale of developing neutralizing antibodies against the S glycoprotein, disrupting its interaction with ACE2 and other receptors and thereby inhibiting viral entry.


###### Spike (S) Neutralizing Antibody Current Evidence

The first human neutralizing antibody against SARS-CoV-2 targeting the trimeric spike (S) glycoproteins has been developed using hybridoma technology, [@doi:10.1101/2020.03.11.987958], where antibody-producing B-cells developed by mice can be inserted into myeloma cells to produce a hybrid cell line (the hybridoma) that is grown in culture.
The 47D11 clone was able to cross-neutralize SARS-CoV and SARS-CoV2 by a mechanism that is different from receptor binding interference.
The exact mechanism of how this clone neutralizes SARS-CoV-2 and inhibits infection in vitro remains unknown, but a potential mechanism might be antibody induced destabilization of the membrane prefusion structure [@doi:10.1101/2020.03.11.987958; @doi:10.1016/j.cell.2018.12.028].
The ability of this antibody to prevent infection at a feasible dose needs to be validated in vivo, especially since in vitro neutralization effects have been shown to not be reflective of in vivo efficacy [@doi:10.1128/JVI.01603-17].
Only a week later, a different group successfully isolated multiple nAbs targeting the RBD of the S glycoprotein from blood samples taken from COVID-19 patients in China [@doi:10.1101/2020.03.21.990770].
Interestingly, the patient isolated antibodies did not cross-react with RBD’s from SARS-CoV and MERS-CoV, although cross-reactivity to the trimeric spike proteins of SARS-CoV and MERS-CoV was observed.
This suggests that the RBDs between the three coronavirus species are immunologically distinct and that the isolated nAbs targeting the RBD of SARS-CoV-2 are species specific.
While this specificity is desirable, it also raises the question of whether these antibodies are more susceptible to viral escape mechanisms.
Viral escape is a common resistance mechanism to nAbs therapy due to selective pressure from neutralizing antibodies [@doi:10.2217/imt.15.33; @doi:10.1126/science.1213256].
For HIV, broadly neutralizing antibodies (bnAbs) targeting the CD4 binding site (CD4bs) show greater neutralization breadth than monoclonal antibodies, which target only specific HIV strains [@doi:10.1016/j.molmed.2019.01.007].
For MERS-CoV, a combination of multiple neutralizing antibodies targeting different antigenic sites prevented neutralization escape [@doi:10.1128/JVI.02002-17].
It was found that the different antibody isolates did not target the same epitopes, suggesting that using them in combination might produce a synergistic effect that prevents viral escape [@doi:10.1101/2020.03.21.990770].
It was also demonstrated that binding affinity of the antibodies does not reflect their capability to compete with ACE2 binding.
Furthermore, no conclusions about correlations between the severity of disease and the ability to produce neutralizing antibodies can be drawn at this point.
Rather, higher neutralizing antibody titers were more frequently found in patients with severe disease.
Correspondingly, higher levels of anti-spike IgG were observed in patients that deceased from infection compared to patient that recovered [@doi:10.1172/jci.insight.123158].

###### Spike (S) Neutralizing Antibody Summary 

Results from the SARS-CoV and MERS-CoV epidemics can provide valuable lessons for the design of neutralizing antibodies for the current outbreak.
The findings for SARS-CoV and MERS can aid in identifying which structures constitute suitable targets for nAbs, despite the fact that the RBD appears to be distinct between the three coronavirus species.
These studies also suggest that a combination of nAbs targeting distinct antigens might be necessary to provide protection [@doi:10.1128/JVI.02002-17].
The biggest challenge remains identifying antibodies that not only bind to their target, but also prove to be beneficial for disease management.
On that note, a recently published study indicates that anti-spike antibodies could make the disease worse rather than eliminating the virus [@doi:10.1172/jci.insight.123158].
These findings underscores our current lack of understanding the full immune response to SARS-CoV-2.

##### Anticipated Mechanism

Why it may be useful

##### Current Evidence

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

##### Summary

Summarize the state of the neutralizing antibody approach.

#### Interferons

Interferons (IFNs) are a family of cytokines crucial to activate the first (innate) immune system response against viral infections.
Interferons are classified into three categories based on their receptor specificity: type I, II and III [@doi:10.1128/mmbr.05015-11].
Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾 ) induce the expression of antiviral proteins which bring the viral RNA to degradation [@doi:10.1016/j.jcv.2003.11.013].
Among these IFNs, IFN- 𝛽 was already found to strongly inhibit the replication of other corona viruses, such as SARS-Cov, in cell culture, while IFN-𝛼 and 𝛾 were shown to be less effective in this context [@doi:10.1016/j.jcv.2003.11.013].
There are evidences that patients with higher susceptibility to develop Acute respiratory distress syndrome (ARDS) show indeed deficiency of IFN- 𝛽.
For instance, upon other Corona viruses infection IFN- 𝛽 expression and synthesis is impaired, so that the virus can in fact escape the innate immune response [@doi:10.1016/j.virusres.2014.07.024].

On March 18 2020 Synairgen plc has received approval to start a phase II trial for SNG001, an IFN- 𝛽-1a formulation to be delivered to lungs via inhalation.
SNG001 was already shown to be effective reducing viral load in swine flu in vivo model, as well as it has been shown to be effective in the protection from other Corona virus infection in vitro (Synairgen plc, press release).

##### Anticipated Mechanism

Why it may be useful

##### Current Evidence

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

##### Summary

Summarize the state of interferons.


### Vaccines

Vaccines, widely recognized as one of the most significant advances in human health during the 20th century, can be used to bolser both individual and herd immunity to a virus by promoting the development of antibodies without infection.
[Are vaccines available for other coronaviruses or related viral illnesses?]
[What are some of the challenges to developing a vaccine? What needs to be taken into account about how the virus works?]
[Are there any challenges or opportunities unique to coronaviruses and/or SARS-CoV-2?]
[What are some approaches being tested or considered?]

#### DNA Vaccines

Brief background on the therapeutic.

##### Anticipated Mechanism

Why it may be useful

##### Current Evidence

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

##### Summary

Summarize the state of the vaccine approach.

#### RNA Vaccines

Brief background on the therapeutic.

##### Anticipated Mechanism

Why it may be useful

##### Current Evidence

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

##### Summary

Summarize the state of the vaccine approach.

#### Viral Particle Vaccines

Brief background on the therapeutic.	

##### Anticipated Mechanism

Why it may be useful	

##### Current Evidence	

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.	

##### Summary

Summarize the state of the vaccine approach.

#### Oligonucleotide Therapies

Background

##### Anticipated Mechanism

Why it may be useful

##### Current Evidence

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

##### Summary

Summarize the state of the neutralizing antibody approach.

### Underexplored Therapeutics

A summary of therapeutic avenues that future research may explore. Possible alternatives to current options and an overview of drug development processes.


## Methods

### Article Selection and Evaluation

The authors solicited relevant articles to be submitted via [GitHub](https://github.com/greenelab/covid19-review) for review.
Articles were classified as _diagnostic_, _therapeutic_, or _other_.
Following a framework often used for assessing medical literature, the review consisted of examining the methods used in the article, the assignment (whether the study was observational or randomized), the assessment, the results, the interpretation, and how well the study extrapolates [@doi:10.5014/ajot.60.4.367].

#### Diagnostic Papers

##### Methods

Reviewers began by describing the study question(s) being investigated by the article.
They then described the study population, the sample size, the prevalence of the disease in the study population, if in human subjects, the countries / regions considered, the demographics of participants, the setting, and any remaining inclusion / exclusion criteria considered.
They then described the reference test or "gold standard," if one was utilized.

##### Assignment

Reviewers described how the new and reference tests were assigned and any further details about the study design, for example whether the diagnostic test was biased towards sicker or healthier individuals or very clear-cut positive/negative cases.

##### Assessment

Reviewers described how the test was performed.
For example, if provided, reviewers described the technical details of the assays used, when measurements were taken and by whom for both the standard and reference diagnostic tests.
They then described how individuals were classified as positive or negative and whether there was evidence that the test results were precise or reproducible when repeated more than once.
Reviewers described whether there was any missing data, whether some participants underwent only one test, or whether there were individuals with inconclusive results.

##### Results

Reviewers reported the estimated sensitivity, specificity, positive predictive value (PPV), and negative predicted value (NPV), as well as the confidence bounds around these measures, if provided.

##### Interpretation

Reviewers reported how well the test ruled in or rules out disease based on the population, if there were identified side effects, and patient adherence.

##### Extrapolation

Reviewers described how well this test will extrapolate outside the measured population.

#### Therapeutic Papers

##### Methods

Reviewers began by describing the study question(s) being investigated by the article.
They then described the study population, the sample size, the prevalence of the disease in the study population, if in human subjects, the countries / regions considered, the demographics of participants, the setting, and any remaining inclusion / exclusion criteria considered.

##### Assignment

Reviewers described how the treatment is assigned, whether it was an interventional or observational study, whether randomization took place, etc.

##### Assessment

###### Outcome Assessment

Reviewers described the outcome that was assessed and evaluated whether it was appropriate given the underlying study question.
They described whether there was any missing data, for example whether there were individuals lost to follow up.
They then describe whether there were any potential sources of bias, for example lack of blinding in a randomized controlled trial.

###### Statistical Methods Assessment

Reviewers describe which statistical methods were used for inference and whether the methods were appropriate for the study.
They then described whether adjustments were made for possible confounders.

##### Results

Reviewers described the estimated association between the treatment and outcome.
They described measures of confidence or statistical significance, if provided.

##### Interpretation

Reviewers described whether a causal claim could be made.
They described whether any side effects or interactions with other drugs were identified, as well as any subgroup findings.

##### Extrapolation

Reviewers describe how the study may extrapolate to a different species or population.

### Collaborative Writing

Crowd-sourced writing with Manubot [@doi:10.1371/journal.pcbi.1007128].


## Additional Items {.page_break_before} 

### Competing Interests

|Author|Competing Interests|Last Reviewed|
|---|---|---|
|Halie M. Rando|None|2020-03-22|
|Casey S. Greene|None|2020-03-22|
|Michael P. Robson|None|2020-03-23|
|Simina M. Boca|None|2020-03-23|
|Nils Wellhausen|None|2020-03-22|
|Ronan Lordan|None|2020-03-25|
|Christian Brueffer|None|2020-03-25|
|Sadipan Ray|None|2020-03-25|
|Lucy D\'Agostino McGowan|None|2020-03-26|
|Anthony Gitter|None|2020-03-26|
|Anna Ada Dattoli|None|2020-03-26|
|Ryan Velazquez|None|2020-04-04|
|John P. Barton|None|2020-04-06|
|Jeffrey M. Field|None|2020-03-30|
|Bharath Ramsundar|None|2020-04-06|
|Adam L. MacLean|None|2020-04-06|

### Author Contributions

|Author|Contributions|
|---|---|
|Halie M. Rando|Project Administration, Writing - Original Draft, Writing - Review & Editing, Methodology|
|Casey S. Greene|Conceptualization, Software|
|Michael P. Robson|Software|
|Simina M. Boca|Methodology|
|Nils Wellhausen|Writing - Original Draft, Writing - Review & Editing|
|Ronan Lordan|Writing - Original Draft, Writing - Review & Editing|
|Christian Brueffer|Writing - Original Draft|
|Sadipan Ray|Writing - Original Draft|
|Lucy D\'Agostino McGowan|Methodology, Writing - Original Draft|
|Anthony Gitter|Methodology|
|Anna Ada Dattoli|Writing - Original Draft|
|Ryan Velazquez|Methodology|
|John P. Barton|Writing - Original Draft, Writing - Review & Editing|
|Jeffrey M. Field|Writing - Original Draft|
|Bharath Ramsundar|Investigation, Writing - Review & Editing|
|Adam L. MacLean|Writing - Original Draft|


## References {.page_break_before}

<!-- Explicitly insert bibliography here -->
<div id="refs"></div>

<!-- Define citation tags below -->
[@tag:Park2020_distancing]: url:https://github.com/parksw3/Korea-analysis/blob/master/v1/korea.pdf
